index.d.ts

```diff
@@ -0,0 +1,25 @@
+import { Plugin } from 'webpack';
+
+interface WebpackChunkHashPluginOptions {
+  /**
+   * A callback to add more content to the resulting hash
+   */
+  additionalHashContent?: (_: any) => string;
```

Comment:
Thank you guys, it was awesome collaboration.

Btw, meant to ask you :) Is it needed for webpack config written in TS or for something else?
Thanks.